posts/2017-10-15-creating-a-serverless-graphql-to-rest-gateway-on-top-of-the-flickr-api.md

```diff
@@ -242,11 +265,13 @@ server.makeReady = (onServerReady) => {
 
 export default server
 ```
-In `server.js` we're defining a custom method `makeReady` on our new Hapi server instance to register our plugins. Normally you would call `server.start()` in the callback for `server.register()`, but instead we're using `server.inject()` to inject the HTTP request event from Lambda, because we're not using Hapi to listen to HTTP events. The reason why we have these two pieces separated this way is that if we called `server.register()` on every invocation of our Serverless event handler, Hapi would throw an error complaining that we've already registered the given plugins. This is because even though your function only executes on invocation, on first invocation the `server` instance that gets created is kept 'frozen' for a short period until either more requests come in and the instance is 'thawed', or a period of time passes in which no requests come in and it is destroyed.
+In `server.js` we're defining a custom method, `makeReady`, on our new Hapi server instance to register our plugins. Normally, you'd want to call `server.start()` in the callback for `server.register()`, but instead we're using `server.inject()` to inject the HTTP request event from Lambda, because we're not using Hapi to listen to HTTP events.
 
-As a general rule of thumb, if your server creates singletons that are expected to live for the duration of the server's uptime, avoid accidentally creating multiple instances of those singletons as a side-effect of your event handler.
+We're separating the two pieces this way because, if we called `server.register()` on every invocation of our Serverless event handler, Hapi would throw an error complaining that we've already registered the given plugins. Even though our function only executes on invocation, on first invocation the `server` newly-created server instance is kept 'frozen' until either more requests come in and the instance is 'thawed', or a period of time passes with no requests and it is destroyed.
```

Comment:
Change 'on first invocation the `server` newly-created server instance' to 'on first invocation the newly-created `server` instance'